Packages.props

```diff
@@ -35,9 +35,9 @@
     <PackageReference Update="Microsoft.AspNetCore.Session" Version="2.2.0" />
     <PackageReference Update="System.IdentityModel.Tokens.Jwt" Version="7.3.1" />
     <PackageReference Update="Okta.AspNetCore" Version="4.5.0" />
-    <PackageReference Update="Mvp.Selections.Client" Version="4.13.1" />
-    <PackageReference Update="StyleCop.Analyzers" Version="1.1.118" />
+    <PackageReference Update="Mvp.Selections.Client" Version="4.14.0" />
+    <PackageReference Update="StyleCop.Analyzers" Version="1.2.0-beta.556" />
```

Comment:
That's so it has the new syntax of C# since there hasn't been a release since 2019 even though there's active development and new beta versions almost every few months